src/components/toast/global_toast_list.tsx

```diff
@@ -283,6 +303,42 @@ export const EuiGlobalToastList: FunctionComponent<EuiGlobalToastListProps> = ({
     );
   });
 
+  if (showClearAllButton && toasts.length > CLEAR_ALL_TOASTS_THRESHOLD) {
+    const dismissAllToastImmediately = () => {
+      toasts.forEach((toast) => dismissToastProp(toast));
+      onClearAllToasts?.();
+    };
+
+    renderedToasts.push(
+      <EuiI18n
+        key="euiClearAllToasts"
+        tokens={[
+          'euiGlobalToastList.clearAllToastsButtonAriaLabel',
+          'euiGlobalToastList.clearAllToastsButtonDisplayText',
+        ]}
+        defaults={['clear all toast notifications', 'Clear all']}
```

Comment:
Super minor copy nit (there isn't any meaningful impact on screen readers, I'm just being annoying)
```suggestion
        defaults={['Clear all toast notifications', 'Clear all']}
```